api/graphql/reservations/reservation_serializers/confirm_serializers.py

```diff
@@ -79,28 +82,32 @@ def validate(self, data):
             payment_type = data.get("payment_type", "").upper()
             reservation_unit = self.instance.reservation_unit.first()
 
-            if self.instance.price_net > 0 and not reservation_unit.payment_product:
-                raise ValidationErrorWithCode(
-                    "Reservation unit is missing payment product",
-                    ValidationErrorCodes.MISSING_PAYMENT_PRODUCT,
-                )
-
-            if not payment_type:
-                data["payment_type"] = self._get_default_payment_type()
-            elif (
-                self.instance.price_net > 0
-                and not reservation_unit.payment_types.filter(
-                    code=payment_type
-                ).exists()
+            active_price = ReservationUnitPricingHelper.get_active_price(

```

Comment:
Oh, just realized this helper class to fetch active pricing. I also added custom QuerySet manager to fetch active pricings in another PR.

Reply:
It probably would be nice to have this inside `ReservationUnit` so that you could just call `reservation_unit.get_active_price()` and there is no need to remember any helper classes  🤔